### PR TITLE
Write multiple files hotfix

### PR DESCRIPTION
**Pull request by TylerSehr**
WriteMultipleFiles times out unless user selects remember when calling the function. 

to reproduce:
- use Cookbook plugin, and select protocols
- click on a protocol
- click accept on the popup, but not tick remember
- call will time out and protocol will not open.

### CODE REVIEW

File: apps/remix-ide/src/app/files/fileManager.ts [hunk 1]
```diff
@@ -227,22 +227,21 @@ class FileManager extends Plugin {
   */
   async writeMultipleFiles(filePaths, fileData, folderPath) {
     try {
-      let alert = true
-      for (let i = 0; i < filePaths.length; i++) {
-        const installPath = folderPath + "/" + filePaths[i]
+      filePaths.forEach(async (filePath, index) => {
```
Comment (joeizang) [inline apps/remix-ide/src/app/files/fileManager.ts:230]: forEach is not a good place to do anything async. It is generally not recommended to do anything async in a foreach because forEach loops doesn't wait for any async ops to finish and the result of which is race conditions and bugs.
Using a for...of or a Promise.all might be better here but your milage may vary.

File: apps/remix-ide/src/app/files/fileManager.ts [hunk 2]
```diff
@@ -227,22 +227,21 @@ class FileManager extends Plugin {
...
+        const installPath = folderPath + "/" + filePath
+        const alert = !index
```
Comment (joeizang) [inline apps/remix-ide/src/app/files/fileManager.ts:232]: You have two variables named the same thing only differentiated by scope which isn't ideal. Your code should communicate intent as much as possible and if your have an inner alert and an outer one then what is the purpose of the outer one.

**Review by joeizang — CHANGES_REQUESTED** [review]

File: apps/remix-ide-e2e/src/tests/compiler_api.test.ts [hunk 3]
```diff
@@ -34,7 +34,15 @@ module.exports = {
       .click('*[data-id="terminalClearConsole"]')
   },
 
-  'Should compile using "compileWithParamaters" API with optimization off check default runs #group3': function (browser: NightwatchBrowser) {
+  'Should compile using "compileWithParamaters" API with optimization off check default runs': function (browser: NightwatchBrowser) {
```
Comment (LianaHus) [inline apps/remix-ide-e2e/src/tests/compiler_api.test.ts:37]: Why was this removed from the group?

**Comment by bunsenstraat** [comment]
It looks to me like lots of changes from some other branch were merged in this one. Beter to start with a clean branch from master and apply this fix there.
